FreeRTOS/FreeRTOS-Partner-Supported-Demos
# Added demo support for dsPIC33A Device

Added demo support for dsPIC33A Device

Description
-----------
Added a new demo for the dsPIC33A device family.
These changes are already deployed in Microchip's repo: https://github.com/microchip-pic-avr-examples/pic24-dspic33-freertos-demo

Test Steps
-----------
1. Build the demo using MPLABX IDE and XC16 or XC-DSC compiler
2. Program the device using the generated hex file
3. Check the functionalities of the demo


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
